src/iio_wrapper.cpp

```diff
@@ -12,7 +12,12 @@ IIOWrapper::IIOWrapper()
 {
     if(m_network_context == nullptr)
     {
+        #ifdef __arm__
```

Comment:
Please remove the #if-def directives. Just use the this: iio_create_context_from_uri("ip:127.0.0.1");
Once you have an implementation that takes an URI from command line, you can replace "ip:127.0.0.1" with the provided URI.

Reply:
I changed the directive wiht uri in another commit.